Sources/Layoutable.swift

```diff
@@ -29,6 +29,9 @@ public protocol Layoutable: AnyObject, Equatable, CustomDebugStringConvertible {
     var superview: PinView? { get }
     var subviews: [PinView] { get }
 
+    var isHidden: Bool { get }
+    var alpha: CGFloat { get }
```

Comment:
I did not find implementation for `NSView` currently (in AppKit this property has different name – `alphaValue`, so it should be explicitly proxied).

Also it looks like a breaking change (adding new requirement to public protocol). Maybe it's fine, but if not, we should have some safe default implementation.

I don't think that we can have reasonable default implementations for `isHidden` and `alpha` because it too general properties, but we can have something like:
```
protocol Layoutable {
  func isVisible(forViewFilter: ViewFilter) -> Bool
  ...
}

extension Layoutable {
  func isVisible(forViewFilter: ViewFilter) { return true } // default implementation
}
```

And then have explicit implementations for UIView, NSView, CALayer:
```
extension UIView {
  func isVisible(forViewFilter viewFilter: ViewFilter) { 
    switch viewFilter {
      case .none: return true
      case .visibleOnly: return !isHidden && alpha > 0
    }
  }
}
...
```

Reply:
Another option to avoid duplicating implementations:

```
protocol Layoutable {
  var isConsideredVisibleForViewFilters: Bool { get }
  ...
}

extension Layoutable {
  var isConsideredVisibleForViewFilters: Bool { return true } // default impl
}

```